.pre-commit-config.yaml

```diff
@@ -46,3 +46,18 @@ repos:
       - id: flake8
         args:
         - --show-source
+  - repo: local
+    hooks:
+      - id: verify-copyright
+        name: verify-copyright
+        description: make sure copyright headers are up to date
+        entry: verify-copyright
+        language: python
+        files: |
+          (?x)
+              [.]py$|
+              [.]pre-commit-config[.]yaml$|
+              [.]pre-commit-hooks[.]yaml$|
+              pyproject[.]toml$
+        args: [--fix, --main-branch=main]
+        additional_dependencies: [.]
```

Comment:
I'm not having success with this, though I expected it would work...

Reply:
FWIW, the other self-referencing pre-commit hook repositories that I could find just use their own public URLs with the latest tags, like this:

https://github.com/pre-commit/pre-commit-hooks/blob/2d948977cf41d3c2b469a33776463c39f3370edb/.pre-commit-config.yaml#L2-L3

Reply:
I initially thought it worked, but it falsely worked because of how my environment was set up. Still investigating.